Formula/p/pivit.rb

```diff
@@ -0,0 +1,18 @@
+class Pivit < Formula
+  desc "Sign and verify data using hardware (Yubikey) backed x509 certificates (PIV)"
+  homepage "https://github.com/cashapp/pivit"
+  url "https://github.com/cashapp/pivit/archive/refs/tags/v0.6.0.tar.gz"
+  sha256 "67a2f353c820aad9c9a4cc937ba256c75cb36044baa0578b74a1bfbdf4e6bd90"
+  license "MIT"
+
+  depends_on "go" => :build
```

Comment:
```suggestion
  depends_on "go" => :build
  depends_on "pkg-config" => :build
```

Reply:
Going to add this to the `on_linux` block only